src/components/Proposals/ProposalDetails/ProposalDetails.js

```diff
@@ -152,7 +158,27 @@ function Details({ vote, status }) {
               margin-top: ${3 * GU}px;
             `}
           >
-            {context}
+            {ipfsJustification ? (
+              <Link
+                href={getIpfsUrlFromUri(ipfsJustification)}
+                css={`
+                  max-width: 90%;
+                `}
+              >
+                <span
+                  css={`
+                    display: block;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    text-align: left;
+                  `}
+                >
+                  {getIpfsCidFromUri(ipfsJustification)}
```

Comment:
Should we show "Read more" instead of the IPFS URI?

Reply:
Done :)